great_expectations/util.py

```diff
@@ -2152,3 +2152,20 @@ def numpy_quantile(
         )
 
     return quantile
+
+
+class NotImported:
+    def __init__(self, message: str):
```

Comment:
I was unsure whether to have default error message or leave it open like this. One may want different error messages depending on if one is importing a module or a symbol from a module.